patient-appointment-management/functions/auth.private.js

```diff
@@ -30,18 +30,12 @@ function createAppToken(issuer, context) {
   });
 }
 
-function createMfaToken(issuer, mfaCode, context) {
+function createMfaToken(issuer, context) {
   if (checkDisableAuthForLocalhost(context)) {
     return createAppToken(issuer, context);
   }
 
-  // encrypt the mfaCode to avoid showing in the browser
-  mfaEncrypt = crypto
-    .createHmac('sha256', context.AUTH_TOKEN)
-    .update(Buffer.from(`${mfaCode}:${context.SALT}`, 'utf-8'))
-    .digest('base64');
-
-  return jwt.sign({ data: mfaEncrypt }, context.AUTH_TOKEN, {
+  return jwt.sign({}, context.AUTH_TOKEN, {
```

Comment:
Calling `jwt.sign()` without a `payload` parameter doesn't make a lot of sense to me; it seems like the values that would typically be in the `payload` are instead being passed as the `options` parameter here.

Reply:
We are using [jsonwebtoken](https://github.com/auth0/node-jsonwebtoken#jwtsignpayload-secretorprivatekey-options-callback) which recommends that `audience` be part of the `options` parameter.

Reply:
Understood. I won't consider this a blocker, but architecturally please consider [OWASP's guidance against using JWTs as a session management token](https://owasp.org/www-chapter-vancouver/assets/presentations/2020-01_Attacking_and_Securing_JWT.pdf).